grafanalib/core.py

```diff
@@ -299,6 +302,61 @@ def TimePicker(refreshIntervals, timeOptions):
 )
 
 
+def AlertCondition(target, queryParams=["A", "5m", "now"],
+                   evaluator=["gt", 5], operator="and", type="avg"):
```

Comment:
Both `evaluator` and `queryParams` look complex enough that they should probably have their own top-level functions, so that we say something like:

```python
AlertCondition(
    target=Target(...), 
    evaluator=GreaterThan(5),
    ...
)
```

I don't really understand what `queryParams` is doing well enough to make a similarly concrete suggestion—what _is_ it for?